tests/glue/jobs/test_s3_to_json.py

```diff
@@ -0,0 +1,60 @@
+import os
+import io
+import json
+import zipfile
+
+import boto3
+import pytest
+
+from src.glue.jobs import s3_to_json
+
+
+class TestS3ToJsonS3:
+    def test_get_metadata_type(self):
+        assert (
+            s3_to_json.get_metadata("HealthKitV2Samples_20201022-20211022.json")["type"]
```

Comment:
Use the actual format like `HealthKitV2Samples_AppleExerciseTime_20230112-20230114.json`